meta-sokol-flex-distro/conf/layer.conf

```diff
@@ -22,3 +22,7 @@ LAYERSERIES_COMPAT_sokol-flex-distro = "kirkstone"
 LAYERDEPENDS_sokol-flex-distro = "core sokol-flex-common"
 LAYERRECOMMENDS_sokol-flex-distro = "sokol-flex-support sokol-flex-staging sourcery flex-private flex-cve flex-vendor-integration"
 LAYERRECOMMENDS_sokol-flex-distro += "openembedded-layer filesystems-layer networking-layer multimedia-layer"
+
+# We have it here because we use upstream virtualization layer without any changes
+INHERIT:append = " feature_overrides"
```

Comment:
the staging layer is doing this already, do we still need this?

Reply:
Thanks. I have removed the INHERIT from here